src/components/SegmentedControl/Option/Option.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import Tooltip from '../../Tooltip';
+import { OptionStyled, ContentWrapper, Label, IconWrapper } from './style';
+
+/**
+ * Option component for Segmented Control component.
+ */
+const Option = (props) => {
+  const {
+    selected,
+    disabled,
+    icon,
+    label,
+    value,
+    optionType,
+    iconPosition,
+    size,
+    tooltip,
+    onClick,
+  } = props;
+
+  // Whether or not to attempt to render an icon
+  const showIcon = optionType === 'icon' || optionType === 'textAndIcon';
+  // Whether or not to render a label
+  const showLabel = optionType === 'text' || optionType === 'textAndIcon';
+
+  const tooltipMessage = useMemo(() => {
+    // If tooltip message is provided, show it
+    if (tooltip) return tooltip;
+    // If no tooltip message is provided, but optionType is 'icon'
+    // derive a tooltip message from the provided label
+    if (optionType === 'icon') return `View ${label} only`;
+  }, [optionType, tooltip, label]);
+
+  const handleClick = () => {
+    // Only run onClick callback if option is not disabled
+    if (!disabled) onClick(value);
+  };
+
+  const renderContent = () => (
+    <ContentWrapper iconPosition={iconPosition}>
+      {icon && showIcon && (<IconWrapper>{icon}</IconWrapper>)}
+
+      {label && showLabel && (
+        <Label optionType={optionType} iconPosition={iconPosition}>
+          {label}
+        </Label>
+      )}
+    </ContentWrapper>
+  );
+
+  return (
+    <OptionStyled
```

Comment:
How would you feel about adding `aria-pressed` to these options to increase accessibility @guldenmw ? I believe screen readers will not be able to tell which option is currently checked without it